examples/basic_org_policies/variables.tf

```diff
@@ -0,0 +1,43 @@
+/**
+ * Copyright 2018 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+variable "organization_id" {
+  description = "The organization id for putting the policy"
+  type        = string
+}
+
+variable "domains_to_allow" {
+  description = "The list of domains to allow users from"
+  type        = list(string)
+}
+
+variable "vms_to_allow" {
+  description = "The list of VMs are allowed to use external IP, every element of the list must be identified by the VM instance name, in the form: projects/PROJECT_ID/zones/ZONE/instances/INSTANCE"
+  type        = list(string)
+  default     = []
+}
+
+variable "exclude_folders" {
```

Comment:
These aren't used in the example, so should they be removed?

Reply:
Right, for the example we don't really need any excludes, Addressed